drivers/display/ls0xx.c

```diff
@@ -103,21 +103,21 @@ static int ls0xx_blanking_on(const struct device *dev)
 
 static int ls0xx_cmd(const struct device *dev, uint8_t *buf, uint8_t len)
 {
-	struct ls0xx_data *driver = dev->data;
+	struct ls0xx_config *config = dev->config;
```

Comment:
```suggestion
	const struct ls0xx_config *config = dev->config;
```
Also a few more occurrences like this below.